app/web/features/search/SearchPage.tsx

```diff
@@ -86,7 +86,9 @@ export default function SearchPage({
     "location"
   );
   const [lastActiveFilter, setLastActiveFilter] = useState(0);
-  const [hostingStatusFilter, setHostingStatusFilter] = useState(0);
+  const [hostingStatusFilter, setHostingStatusFilter] = useState<
+    Exclude<HostingStatus, HostingStatus.HOSTING_STATUS_UNKNOWN>[] | undefined
```

Comment:
The `Exclude` is because in the `Select` options in `FilterDialog` you're not including the `HostingStatus.HOSTING_STATUS_UNSPECIFIED` value.

If you don't want them to be able to choose unspecified, you need to exclude it from the type for `hostingStatusFilter` as well.

Reply:
Second question though, since that's a value that might be saved in the DB, do you really want to exclude it? Maybe people want to search for people who did not specify their hosting status? Or is it required now?

Reply:
it's not saved in the DB, since the queries are generated dynamically, this is a SELECT in SQL, not an UPDATE, DELETE or any other write operation

Reply:
so `HostingStatus.HOSTING_STATUS_UNSPECIFIED` would be (almost) the same as not filtering by hosting status at all

Reply:
so from the UI perspective you are offering a option to the user, which IMHO would confuse him more rather than giving him an useful filter

Reply:
Ah no I don't mean the Select value is stored in the DB. I mean the hostingStatus is stored in the DB in the `users` table.

What if the person us searching for someone to meet up with or my username and they don't care about their hosting status?